functions.yml

```diff
@@ -432,3 +454,4 @@ gwas_functions:
       args:
         - name: genotypes
           doc: An array of genotype structs
```

Comment:
An array of genotype structs with ``calls`` field

Reply:
Done